src/output/netcdf_output.jl

```diff
@@ -201,7 +201,7 @@ function add_default!(
     Model::Type{<:PrimitiveDry},
 )
     add_default!(variables, Barotropic)
-    add!(variables, SurfacePressureOutput(), TemperatureOutput())
+    add!(variables, MeanSeaLevelPressureOutput(), TemperatureOutput())
```

Comment:
This makes MSLP now default output for primitive models instead of surface pressure which is too dominated by orography to be meaningful dynamically